Home.md

```diff
@@ -17,3 +17,57 @@ Use the [`#WindowsCommunityToolkit`](https://twitter.com/search?q=%23WindowsComm
 - Windows Blogs - Release Overviews - [Search for Windows Community Toolkit](https://blogs.windows.com/?s=Windows+Community+Toolkit)
 - Dev Blogs - [Other Articles](https://devblogs.microsoft.com/pax-windows/)
 - We may sometimes add to the [.NET Foundation Newsletter](https://github.com/dotnet-foundation/website/blob/master/input/blog/posts/_current-newsletter-draft.md) too!
+
+## Projects using the Windows Community Toolkit
+
+Below are a few projects highlighting usage of the Windows Community Toolkit from other creators in our Community. We hope they inspire you to see the possibilities of what you're able to create with the tools provided in our library.
+
+### [Fiona](https://www.mysqueezebox.com/download) by [PaoloM](https://github.com/PaoloM)
+
+![Fiona1](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Fiona1.png)
```

Comment:
Let's clean-up the images below in the image folder as a sub-folder, and for only the one's we're using (images/app-gallery), then we can reference them relatively like:

`![Expanded PR Checks with Build Number](images/Preview-Packages-PR-Checks.png)`

Let's stick with pngs or jpgs vs large gifs as we don't want a ton of images in the git repo.